doc/source/cluster/vms/references/ray-cluster-configuration.rst

```diff
@@ -953,7 +953,14 @@ The user that Ray will authenticate with when launching new nodes.
 
     .. tab-item:: Azure
 
-        Not available.
+        For Azure, the avability zone avability depends on each specific VM size / location combination.
```

Comment:
So for azure the availability zones config is not at the cluster level but at each node group level. Do we need that level of granularity? AWS and GCP specify at the provider level.

Reply:
In Azure availability zone config depends on the VM size _and_ region.
Since each node group can specify a different VM size, we should specify the possible availability zones for each node group.

Reply:
Do we throw error if user specifies `provider.availability_zone`?

Reply:
agree.

Reply:
I think for cross-provider compatibility it will be valuable to expose a sane `provider.availability_zone` value, as a user coming from e.g., GCP may try to copy/paste well-known settings from another provider and expect it to "just work".

+1 on making the default `provider.availability_zone` value null/empty

Reply:
sounds good, let me make these updates

Reply:
I'm working on these updates today